config/install/system.site.yml

```diff
@@ -4,7 +4,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /node/1
+  front: /node/123321
```

Comment:
Can we just have empty or something?

I think that we're reinventing the wheel... The best way is fixing issue with redirect and use aliases. I'm just thinking of amount of code that we should maintain on order just to have workaround for redirects module.